shortly/app/__init__.py

```diff
@@ -0,0 +1 @@
+from .main import app  # noqa
```

Comment:
Is "noqa" necessary?
[tool.flakeheaven.exceptions."**/__init__.py"]
pyflakes = ["-F401", "-F403"]